extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -134,3 +134,16 @@ export function doubleclickIsA4AEnabled(win, element) {
   }
   return enableA4A;
 }
+
+export const adConfig = {
```

Comment:
renderStartImplemented gets used in XoriginIframeHandler, which gets called from within the A4A impl so we need that too. Everything else can go.